reana_commons/openapi_specifications/reana_workflow_controller.json

```diff
@@ -158,6 +162,9 @@
                       "id": {
                         "type": "string"
                       },
+                      "launcher_url": {
+                        "type": "string"
```

Comment:
I got an error saying that `null` values are invalid, since only `string`'s allowed here. After a quick lookup I found that `nullable` types are not supported by openapi v2 spec, more about it [here](https://jane.readthedocs.io/en/latest/tips/nullable.html#openapi-v2-swagger). So a quick workaround would be to add `"x-nullable": true` here, it is supported by `bravado` and works (just checked)